src/sourmash/command_sketch.py

```diff
@@ -38,8 +38,14 @@ def _parse_params_str(params_str):
                 num = int(num)
             except ValueError:
                 raise ValueError(f"cannot parse num='{num}' as a number")
+            # if num < 0:
+            #     raise ValueError(f"num is {num}, must be >= 0")
             if num < 0:
-                raise ValueError(f"num is {num}, must be >= 0")
+                raise ValueError(f"ERROR: --num-hashes value must be positive")
```

Comment:
but --num-hashes isn't used in param strings, so these error messages are problematic, aren't they?

Reply:
maybe change the error messages so they refer to num? not sure. "Number of hashes must be >= 0" would maybe be ok.

Reply:
Yes, I thought that would be a problem. I can just change it to: `ERROR: --num value must be positive` if that's okay.